src/demos/DlSliderDemo.vue

```diff
@@ -58,6 +58,14 @@ export default defineComponent({
         handleChange(value: number) {
             console.log(`@@@ handling change ${value}`)
             this.events[1] = `@@@ handling change ${value}`
+        },
+        handleFocus(event: Event) {
+            console.log(`@@@ handling focus ${event}`)
```

Comment:
Notice to remove debugging and console logs from the code

Reply:
removed the console logs